client/post-editor/editor-html-toolbar/style.scss

```diff
@@ -85,6 +85,10 @@
 		width: 0;
 	}
 
+	&.has-active-drop-zone {
+		position: static; /* Allow the dropzone to overflow the toolbar and expand to fill the entire editor body */
```

Comment:
I was a little worried about this, just in case some element relied on it's `relative` positioning but it seems that it's direct child `editor-html-toolbar__wrapper` is relative, and so this should be fine.
I've not dealt with `position: static;` much before so I'm not 100% confident of it's use - might be worth borrowing some CSS pro 👀s for this maybe?

Reply:
Oh, I think [it's just the default value](https://developer.mozilla.org/en-US/docs/Web/CSS/position). The reason I chose it for this situation is because it doesn't make the toolbar count as a "positioned" element, so it wont set the [positioning context](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Positioning) for the dropzone.

If `.editor-html-toolbar` used `relative`,  `absolute`, etc, then it would set the positioning context for its children, and they'd be positioned in relation to it. So, the dropzone be confined to the `39px` height of the toolbar.

Because the toolbar is `static` in this situation, the browser will position the dropzone in relation to the next positioned element it encounters as it walks up the DOM tree, which in this case is `.tinymce-container`.

At least, that's my understanding of it. I wouldn't be surprised if I got some of the particulars wrong :)

Reply:
After writing all that, I realize I probably should have documented that in the actual code. I'll add a commit with a more succinct explanation :)